app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,47 @@
+class GroupsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_group, only: [:show, :edit, :update, :destroy]
```

Comment:
I think this is fine as is -- @adarsh has a valid point, but since it's not going to be terribly difficult to figure out where @group is set since we're seeing the callback at the top of the file, I think it's fine.

I prefer not to use the memoized version suggested [here](https://github.com/greencommons/commons/pull/66/files#r91846317) since it's not completely obvious at the top of the file, which is what this implementation allows.

This is good as is in my book but if anyone feels strongly I'm totally open to go with their choice.